soh/src/overlays/actors/ovl_En_Elf/z_en_elf.c

```diff
@@ -971,7 +971,7 @@ void func_80A03CF8(EnElf* this, PlayState* play) {
                         func_80A03148(this, &nextPos, 0.0f, 20.0f, 0.2f);
                         EnElf_SpawnSparkles(this, play, 16);
                     } else {
-                        if (distFromLinksHead > 100.0f) {
+                        if (distFromLinksHead > 100.0f) { // if Navi is too far away, she flies back.
```

Comment:
I didn't intend to commit these comments and will push another commit that reverses this soon.